installation/language/en-GB/joomla.cli.ini

```
@@ -0,0 +1,34 @@
+INSTL_SITE_NAME_DESC="Enter the name of your Joomla site."
+INSTL_SITE_NAME_DESC_SHORT="Name of the website"
+INSTL_ADMIN_USER_DESC="Enter the real name of your Super User. "
+INSTL_ADMIN_USER_DESC_SHORT="Real name of the Super User account"
+INSTL_ADMIN_USERNAME_DESC="Set the username for your Super User account."
+INSTL_ADMIN_USERNAME_DESC_SHORT="Username of your Super User account"
+INSTL_ADMIN_PASSWORD_DESC="Set the password for your Super User account."
+INSTL_ADMIN_PASSWORD_DESC_SHORT="Password of your Super User account"
+INSTL_ADMIN_EMAIL_DESC="Enter the email address of the website Super User."
+INSTL_ADMIN_EMAIL_DESC_SHORT="Email address of the websites Super User account"
```

```suggestion
INSTL_ADMIN_EMAIL_DESC_SHORT="Email address of your Super User account."
```

consistency with other strings